WordPress/gutenberg
# Include alignwide in nested has-outer-padding logic

## What?
Removes the omission of `alignwide` from the outer padding logic. Closes https://github.com/WordPress/gutenberg/issues/63008. 

## Why?
The original effort in https://github.com/WordPress/gutenberg/pull/60715 was intended to connect the "Inner blocks use content width" control with applying global padding to the element, reducing the logic of when global padding was applied.  

While this omits the wide aligned + constrained blocks from applying global padding, all other instances remain. 

## How?
Removes the targeting of alignwide. 

## Testing Instructions

1. Activate Twenty Twenty Four.
2. Open the Site Editor.
3. View the "An array of resources" pattern on the front page. 
4. See the padding not changed from 6.5 to 6.6 RC. 

## Screenshots or screencast 


| Before  | After |
| ------------- | ------------- |
|![CleanShot 2024-07-05 at 14 18 47](https://github.com/WordPress/gutenberg/assets/1813435/965eadee-940c-42f8-94d5-bad93553a1dc)|![CleanShot 2024-07-05 at 14 19 13](https://github.com/WordPress/gutenberg/assets/1813435/6e11fefc-9ed8-46f9-a544-b3eab1a2c67e)|
